packages/core/src/lib/TwaManifest.ts

```diff
@@ -155,6 +157,10 @@ export class TwaManifest {
   alphaDependencies: alphaDependencies;
   enableSiteSettingsShortcut: boolean;
   isChromeOSOnly: boolean;
+  isMetaQuest: boolean;
+  scope: string | undefined;
+  fullScopeUrl?: URL;
+  minSdkVersion: number;
```

Comment:
@andreban Please check these 4 lines above carefully. What of them should be optional and what can be undefined?

Reply:
An option would be using the Manifest URL to generate a default scope and fullScopeUrl, if we want to prevent undefined values. But I'm happy either way.

Reply:
@andreban Is there a case when there is no Manifest URL?

Reply:
Yes, we didn't save that information in the past.

Reply:
@andreban I mean are there any ways to generate `twa-manifest.json` file without CLI wizard? If we use CLI wizard we must provide `--manifest` parameter. So we cannot get into an invalid state.

As an example, what if I open the `twa-manifest.json` file in a text editor and edit it so that `"isMetaQuest": true`, but `"webManifestUrl"` is missing? Should we care about such cases?

Can we get into an invalid state by updating the old `twa-manifest.json` (using `merge` command on the old `twa-manifest.json`)?